l10n_it_ddt/wizard/ddt_create_invoice.py

```diff
@@ -13,13 +13,17 @@ def _get_ddt_ids(self):
         return self.env['stock.picking.package.preparation'].browse(
             self.env.context['active_ids'])
 
+    include_down_payment_invoice = fields.Boolean(
+        'Include Down Payment Invoice')
```

Comment:
Non sarebbe più corretto "Exclude"? Altrimenti "subtract"

Reply:
Avere include ci permette di far funzionare tutto il codice anche di altri eventuali moduli senza dover toccare niente. Infatti questo valore lo sfrutto a livello di context così gli sviluppi fatti su questo modulo non vanno a rompere roba regressa.

Reply:
> far funzionare tutto il codice anche di altri eventuali moduli senza dover toccare niente

Non mi è chiaro, ad esempio quali moduli?

Reply:
In generale. Così come è scritto ora il modulo nella PR, tutte le chiamate alla funzione di generazione della fattura continuano a funzionare come ci si aspetta. Pertanto bisogna indicare espressamente di voler includere la fattura di anticipo

Reply:
Non sto suggerendo di cambiare il comportamento, solo il termine "include". Mi sembra che "exclude" o "subtract" sia più in linea con ciò che avviene: sottrarre (o escludere), in fattura, quanto già fatturato per l'anticipo

Reply:
@eLBati scusami, avevo capito che intendessi invertire il comportamento di quella voce. Ora mi è più chiaro.